### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by LovelyDrug**
[DEMO LINK](https://LovelyDrug.github.io/react_keyboard/)

### CODE REVIEW

**Review by opanasiukvlad — APPROVED** [review]
Good job!

File: src/App.tsx [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleClick);
+  }
+
+  handleClick = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
```
Comment (VETALEK) [inline src/App.tsx:21]: let's use `event.code` to check if the `space` is pressed and write `[space]` if so
image

**Review by VETALEK — CHANGES_REQUESTED** [review]
GJ 👍 
commented

File: src/App.tsx (hunk 1 above)
Comment (ivan-kadykalo) [reply src/App.tsx:21]: Fix this comment
image

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Fix comments
image

**Comment by LovelyDrug** [comment]
>

**Pull request closed without merging**